docs/adr/0003-uri-variables.md

```diff
@@ -0,0 +1,73 @@
+# URI Variables
+
+* Status: accepted
+* Deciders: @dunglas @soyuka
+
+Implementation: [#4408][pull/4408]
+
+## Context and Problem Statement
+
+For reference see the previously implemented [Resource identifier](0001-resource-identifiers) ADR. 
+URI variables are the URI template (e.g. `/books/{id}`) variables (e.g. `id`). When defining alternate resources (see [Resource definition](0002-resource-definition)), we need more informations then only a property and a class. The tuple accepted in [Resource identifier](0001-resource-identifiers) is not enough. For example:
```

Comment:
```suggestion
URI variables are the URI template (e.g. `/books/{id}`) variables (e.g. `id`). When defining alternate resources (see [Resource definition](0002-resource-definition)), we need more information than only a property and a class. The tuple accepted in [Resource identifier](0001-resource-identifiers) is not enough. For example:
```

Reply:
thx fixed in https://github.com/api-platform/core/pull/4441